flexmeasures/data/models/planning/utils.py

```diff
@@ -182,6 +183,7 @@ def get_power_values(
     :param resolution:      timedelta used to resample the forecasts to the resolution of the schedule
     :param beliefs_before:  datetime used to indicate we are interested in the state of knowledge at that time
     :param sensor:          power sensor representing an energy flow out of the device
+    :param default_value:   value to fill in missing values
```

Comment:
Please add a comment about the interpretation of the sign of this value. This function returns consumption as positive values, but currently, the default value should be given corresponding to the sign convention for the given sensor, as defined by the sensor's `consumption_is_positive` attribute.

I think it would be more intuitive to have the sign convention for the default value correspond to the sign convention of the return values. In that case, the `fillna` statement should move after the sign conversion based on the `consumption_is_positive` attribute.

Reply:
You're right! Nonetheless, I've end up using just part of the function on another new function.